modules/preview-react/InformationHighlight/lib/parts/Heading.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import {createComponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {createStyles, handleCsProp} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
+import {Heading} from '@workday/canvas-kit-react/text';
+
+const headingStyles = createStyles({
```

Comment:
We should rename this styles so that're more specific to the components. When the css gets generated their could be multiple headingStyles. We can rename this to something like `informationHighlightHeadingStencil`. We should do the same for the other parts